content/get_start/UseCase_Workflows/pollination.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "PRM for Pollination"
+date: 2022-09-21T15:00:28-07:00
+weight: 252
+draft: false
+pre: "<b>- </b>"
+---
+
+{{<line_break>}}
+
+#### B. Use Case 2: Application of the PRM measure to a user-model created in Pollination
+
+This use case showcases how an energy modeler can utilize a BIM (Building Information Modeling) model created in Pollination by an architect/engineer and can translate it to a BEM (Building Energy Modeling) model to apply the PRM measure in OpenStudio. The goal is to automatically generate a proposed and one or four baseline models according to [ASHRAE 90.1 PRM rules](/BEM-for-PRM/overview/ashrae) to streamline the energy modeling process. 
+
+[Pollination](https://www.pollination.cloud/) is a complete building performance simulation platform that is used to generate models, run simulations, and deliver results. It has a [Revit plugin](https://www.pollination.cloud/revit-plugin) that helps extract analytical models directly from the design model to get real-time insight on energy consumption, daylight and thermal comfort. The workflow provided is based on using the Revit Plugin for Pollination as a stand-alone application. Additionally, please note that the steps provided are based on the capabilities present in the software at the time of writing. Changes and updates to the software may lead to modifications in these steps over time. 
+
+The steps followed for this use case are as follows:
+
+1. [**Create a model**](#1-create-a-model)
+2. [**Define the Construction Set, Program type, HVAC**](#2-define-the-construction-set-program-type-hvac)
+3. [**Export the model**](#3-export-the-model)
+4. [**Apply 90.1 PRM method**](#4-apply-901-prm-method)
+
+{{< line_break >}}
+
+##### **1. Create a model**
```

Comment:
Create a design model (distinguish it from the term proposed and baseline) - same thing can apply to Revit approach.

Reply:
Done!